gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/orc/OrcKeyDedupReducer.java

```diff
@@ -59,8 +59,14 @@ protected void reduce(OrcKey key, Iterable<OrcValue> values, Context context)
     /* Map from hash of value(Typed in OrcStruct) object to its times of duplication*/
     Map<Integer, Integer> valuesToRetain = new HashMap<>();
     int valueHash = 0;
+    int counter = 0;
```

Comment:
This reduce method will be called once for each key,  which means if you define count here, you probably never see the log and it requires 1000 records for same key which is highly not possible. 
You might want to define the count at class level

Reply:
moving to class level so it's overall count